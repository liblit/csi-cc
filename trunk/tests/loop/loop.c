#include <stdio.h>

int rand(){
  static int x = 3; 
  return(x = (x * 8121 + 28411) % 134455);
}

int main(){
  int x = rand()%14;
  while(x!=2){
    printf("ANSWER: %d\n", x);
    int y = rand()%2;
    if(y==1){
      printf("Y= %d\n", 1);
    }
    else
      printf("Y= %d\n", 0);
    x = rand()%14;
  }
  printf("DONE: %d\n", x);
}
