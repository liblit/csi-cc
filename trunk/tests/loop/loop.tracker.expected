#
rand
1|EXIT
0|ENTRY|5|5|5|5|5|5|5|5|-1|5
$
0->1|0$0
#
main
3|EXIT
2|ENTRY|9|9|9|9|9|9|9
4|10|10|10
5|11|11|12|12|12|12|13|13|13
6|-1|20|20|21|21
7|14
8|17
9|18|18|18|-1
$
2->4|0$0
4->5|0$0
4->6|2$2
5->7|0$0
5->8|1$1
6->3|0$0
7->9|0$0
8->9|0$0
9~>4|3$3
